src/services/__tests__/schema.test.tsx

```diff
@@ -22,15 +22,14 @@ describe('schema service', () => {
     test('returns the right properties list', () => {
       const ontology = SCHEMA_DEFN.get('Ontology');
       const testProps = testSchema.getProperties('Ontology');
-
       testProps.forEach((prop) => {
         expect(ontology.properties[prop.name]).toBeDefined();
       });
     });
 
     test('Returns edges', () => {
-      const edges = Object.values(SCHEMA_DEFN.schema)
-        .filter((model) => model.inherits && model.inherits.includes('E'))
+      const edges = Object.values(SCHEMA_DEFN.models)
+        .filter((model) => SCHEMA_DEFN.ancestors(model.name) && SCHEMA_DEFN.ancestors(model.name).includes('E'))
```

Comment:
I don't think the first condition is necessary. looking at the code `ancestors` always returns a list so the first condition will always be true https://github.com/bcgsc/pori_graphkb_schema/blob/v4.0.0/src/schema.ts#L248-L257